app/app.go

```diff
@@ -37,7 +38,7 @@ type App struct {
 	Cluster          einterfaces.ClusterInterface
 	Compliance       einterfaces.ComplianceInterface
 	DataRetention    einterfaces.DataRetentionInterface
-	Elasticsearch    einterfaces.ElasticsearchInterface
+	SearchEngine     *searchengine.SearchEngineBroker
```

Comment:
This is technically a breaking change. I don't have any issues with it. Just wanted to point it out if PMs want to mention it in the release notes.

Reply:
The APP layer is an internal API so It shouldn't be used from anything outside mattermost-server code.

Reply:
Ah okay :+1: